gax/src/main/java/com/google/api/gax/batching/Batcher.java

```diff
@@ -79,4 +81,10 @@
    */
   @Override
   void close() throws InterruptedException;
+
+  /**
+   * Closes this Batcher by preventing new elements from being added and send outstanding elements.
```

Comment:
and sending?

unclear: does it prevent new elements from being added and prevents outstanding elements from being sent 

or does it

send outstanding elements and prevent new elements from being added?

Reply:
I updated the docs, it should make it clearer (i hope)